sonar_topic: "/sonar_array"
motor_topic: "/stepper_array"
side_tolerance: 20
front_tolerance: 13
turn_steps: 1150